Info-Orbs/include/widgets/stockScreenWidget.h

```diff
@@ -0,0 +1,34 @@
+#ifndef STOCK_SCREEN_WIDGET_H
+#define STOCK_SCREEN_WIDGET_H
+
+#include "Widget.h"
+#include <vector>
+
+class StockScreenWidget : public Widget {
+public:
+    StockScreenWidget(ScreenManager &manager);
+    void setup() override;
+    void draw() override;
+    void update() override;
+
+private:
+    void getStockData();
+    void displayStock(int displayIndex, uint32_t backgroundColor, uint32_t textColor);
+
+    unsigned long m_stockDelayPrev;
+    unsigned long m_stockDelay; // Add this line
+    int currentStockIndex;
+    std::vector<String> stockTickers;
+
+    // Stock data variables
+    String m_TrackingStock;
+    String currentPrice;
+    String change;
+    String percentChange;
+    String highPrice;
+    String lowPrice;
+    String openPrice;
+    String previousClosePrice;
```

Comment:
I believe the pattern is to prefix all private member variables with `m_`. I know you were copying what was in the weatherWidget but it is wrong and will be changed in the near future.

Reply:
Good to know... I have no problem with this.